src/test/resources/alma-fix/990050000600206441.json

```diff
@@ -266,11 +266,11 @@
     "type" : [ "Item", "PhysikalischerTitel" ],
     "currentLocation" : "B4002 / UNASSIGNED",
     "heldBy" : {
-      "id" : "http://lobid.org/organisations/DE-294-22#!",
-      "isil" : "DE-294-22",
-      "label" : "Ruhr-Universität Bochum, Verbundbibliothek IB Fachbibliothek für Geographie, Geologie, Mathematik, Psychologie"
```

Comment:
https://sigel.staatsbibliothek-berlin.de/suche?hit=3&q=DE-294-22%20&sort=sortby%20isl%2Fsort.ascending

Not sure why this is kicked out.

Reply:
Ah, the owner does not exist anymore. I could use the replacement isil instead.